packages/cordova-plugin-aerogear-metrics/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@aerogear/cordova-plugin-aerogear-metrics",
+  "version": "0.0.1",
```

Comment:
[Info] We going to keep the versions the same. No need to change as lerna will do that.